.kubernetes/kubernetes_deployments.yaml

```diff
@@ -0,0 +1,88 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: fakebook-auth-api
+  labels:
+    app: fakebook-auth
+    role: api
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: fakebook-auth
+      role: api
+  strategy:
+    rollingUpdate:
+      maxSurge: 3
+      maxUnavailable: 1
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: fakebook-auth
+        role: api
+    spec:
+      containers:
+      - name: api
+        image: fakebook.azurecr.io/fakebookauth:latest
+        ports:
+        - containerPort: 80
+        env:
+        - name: ConnectionStrings__FakebookAuth
+          value: 'Host=fakebook-auth-db;Database=postgres;Username=postgres;Password=Pass@word'
+        envFrom:
+        - secretRef:
+            name: fakebook-auth-config
```

Comment:
i'll create this secret on the cluster but it'll be empty until i hear otherwise